src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowSvcTest.java

```diff
@@ -272,12 +276,38 @@ void testStandByAutomaton() {
                 .withStandbyStatus(true)
                 .add();
 
+        parametersExt.setSvcVoltageMonitoring(true);
         LoadFlowResult result = loadFlowRunner.run(network, parameters);
+        assertTrue(result.isFullyConverged());
 
         assertReactivePowerEquals(150.091, svc1.getTerminal());
         assertVoltageEquals(387.415, bus2);
     }
 
+    @Test
+    void testStandByAutomatonAndSlope() {
+        svc1.setVoltageSetpoint(385)
```

Comment:
Is there a test in which the StandByAutomaton is not defined while the VoltagePerReactivePowerControl is?

Reply:
Yes, see the testSvcWithSlope till testSvcWithSlope5 tests (were already existing)
Same, the tests testStandByAutomaton till testStandByAutomaton5 were already existing.

I added this test for the case where both extensions are defined - did not exist before. Today's behavior is that StandByAutomaton takes priority.

Reply:
ok thanks!